editor/project.clj

```diff
@@ -163,7 +163,9 @@
                       ;; hide warnings about illegal reflective access by clojure
                       "--add-opens=java.xml/com.sun.org.apache.xerces.internal.jaxp=ALL-UNNAMED"
                       ;; used in editor.scene$read_to_buffered_image
-                      "--add-opens=java.desktop/sun.awt.image=ALL-UNNAMED"]
+                      "--add-opens=java.desktop/sun.awt.image=ALL-UNNAMED"
+                      ;; "-XX:MaxJavaStackTraceDepth=1073741823"
+                      ]
```

Comment:
II think it's nice to keep it here for such cases, to know how to increase stack to be able to debug such stack overflow issues